src/main/java/com/zendesk/maxwell/schema/ddl/SchemaChange.java

```diff
@@ -18,21 +20,36 @@
 
 	public abstract Schema apply(Schema originalSchema) throws SchemaSyncError;
 
+	private static final Set<String> SQL_BLACKLIST = new HashSet<String>();
+	static {
+		SQL_BLACKLIST.add("BEGIN");
+		SQL_BLACKLIST.add("COMMIT");
+
+	}
+
 	public static List<SchemaChange> parse(String currentDB, String sql) {
-		ANTLRInputStream input = new ANTLRInputStream(sql);
-		mysqlLexer lexer = new mysqlLexer(input);
-		CommonTokenStream tokens = new CommonTokenStream(lexer);
-		mysqlParser parser = new mysqlParser(tokens);
+		if ( SQL_BLACKLIST.contains(sql))
+			return null;
```

Comment:
I'm not totally sure, but I think that for transactions that touch multiple tables you'll see a `BEGIN` at the the top of them; you have to keep things atomic from the slaves' perspective too... ? 

I don't actually know.